File: greetings.cpp
#include <iostream>
#include <iomanip>
using namespace std;
int main(){
    string m,concat="h";
    int count = 0;
    bool check = true;
    cin>>m;
     for(int i=0;i<m.length() && check;i++){
       if(m[i]=='e'){
         count=count+1;
       }
       else if(m[i]=='y'){
          check = false;
       }
     }
     for(int j=0;j<count*2;j++){
        
       concat = concat + "e";
     }
     cout<<concat+"y";  
  }

File: Greetings.java
import java.util.*;

class Main {
  public static void main(String[] args) {
    Scanner kbd = new Scanner(System.in);
    String m=kbd.nextLine();
    int count = 0;
    String concat= "h";
    boolean check = true;
     for(int i=0;i<m.length() && check;i++){
       if(m.charAt(i)=='e'){
         count=count+1;
       }
       else if(m.charAt(i)=='y'){
          check = false;
       }
     }
     for(int j=0;j<count*2;j++){
        
       concat = concat + "e";
     }
     System.out.print(concat+"y");  
  }
}
